UKSI/scripts/35 - cleanup.sql

```diff
@@ -74,6 +87,23 @@ FROM uksi.preferred_name_changes nc
 WHERE nc.old_taxon_meaning_id=ta.to_taxon_meaning_id
 AND ta.to_taxon_meaning_id<>nc.new_taxon_meaning_id;
 
+
+DELETE FROM cache_taxon_paths WHERE taxon_meaning_id IN (
```

Comment:
Why does this query not filter out any which are linked to new_taxon_meaning_Id (like the commented out query below)? Is it because we need a more thorough rebuild?

Reply:
I think this is a mistake John. I think it's left over from me trying to resolve a problem. I'll remove it before trying the new run on restored dev warehouse.

Reply:
On closer inspection, it's not a mistake. It's the result of the conversation we had here: https://github.com/BiologicalRecordsCentre/iRecord/issues/636#issuecomment-520751086.

> So, I wonder if the easiest way forward is to replace the update statement above with a delete statement (just to remove any pointing to old taxon meanings), then hope that the cache update scripts do their thing and recreate any missing CTP records + update existing ones.